src/robusta/core/sinks/slack/templates/template_loader.py

```
@@ -0,0 +1,125 @@
+                    blocks.append(block)
+
+                except json.JSONDecodeError as e:
+                    logging.exception(f"Error parsing JSON from template output: {e}")
+                    logging.warning(f"Problematic JSON (repr): {repr(block_str)}")
+
+            return blocks
+        except Exception as e:
+            logging.error(f"Error rendering template {template_name}: {e}")
+            return []
+
+    def render_custom_template_to_blocks(self, custom_template: str, context: Dict[str, Any]) -> List[Dict[str, Any]]:
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)